src/Admin.php

```diff
@@ -25,6 +25,7 @@ public static function menu() {
   public static function init() {
     register_setting('shop-analytics-settings', 'shop_analytics_gtm_id');
     register_setting('shop-analytics-settings', 'shop_analytics_gtm_embed');
+    register_setting('shop-analytics-settings', 'shop_analytics_tc_enabled');
```

Comment:
IMO it would be good to be a bit more expressive with the naming.
```suggestion
    register_setting('shop-analytics-settings', 'shop_analytics_tag_commander_enabled');
```